client/templates/sjo-tasks/uncomplete-service/select-jurors.njk

```diff
@@ -32,13 +32,10 @@
       
         <h1 class="govuk-heading-xl">Uncomplete service</h1>
         
-        <table>
-          <caption class="govuk-table__caption govuk-visually-hidden">Uncomplete service search</caption>
-          <tr>
-            <td><h2 class="govuk-heading-m">Showing results for “{{ searchTerm }} ”</h2></td>
-            <td><div class="govuk-body"><a class="govuk-link" href="{{ changeUrl }}" aria-label="Change search term">change</a></div></td>
-          </tr>
-        </table>
+        <h2 class="govuk-heading-m">
+          Showing results for “{{ searchTerm }}”
+          <a class="govuk-link" href="{{ changeUrl }}" aria-label="Change search term">change</a>
```

Comment:
add 'govuk-body' to the classes for the change link, otherwise far too big

Reply:
Thanks, I've added the class